packages/common/src/data/location_data.json

```
@@ -1,9 +1,9 @@
 {
   "has": {
     "byId": {
-      "VancouverRegion": {
-        "id": "VancouverRegion",
-        "name": "Vancouver Region",
+      "VancouverCostal": {
+        "id": "VancouverCostal",
```

Looks like a typo in a few places
```suggestion
        "id": "VancouverCoastal",
```